src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -0,0 +1,658 @@
+        this._categoryClassName = this._buildClassnameForFlightCategory(aircraftModel);
+
+        return this;
+    }
+
+    /**
+     * Set initial element references
+     *
+     * Should be run once only on instantiation
+     *
+     * @for StripViewModel
+     * @method _createChildren
+     * @chainable
+     */
+    _createChildren() {
+        this.$element = $(STRIP_VIEW_TEMPLATE);
+        this.$callsignView = this.$element.find(STRIP_VIEW_SELECTORS.CALLSIGN);
+        this.$transponderView = this.$element.find(STRIP_VIEW_SELECTORS.TRANSPONDER);
+        this.$aircraftTypeView = this.$element.find(STRIP_VIEW_SELECTORS.AIRCRAFT_MODEL);
+        this.$assignedAltitudeView = this.$element.find(STRIP_VIEW_SELECTORS.ASSIGNED_ALTITUDE);
+        this.$flightPlanAltitudeView = this.$element.find(STRIP_VIEW_SELECTORS.FLIGHT_PLAN_ALTITUDE);
+        this.$arrivalAirportView = this.$element.find(STRIP_VIEW_SELECTORS.ARRIVAL_AIRPORT_ID);
+        this.$departureAirportView = this.$element.find(STRIP_VIEW_SELECTORS.DEPARTURE_AIRPORT_ID);
+        this.$alternateAirportView = this.$element.find(STRIP_VIEW_SELECTORS.ALTERNATE_AIRPORT_ID);
+        this.$flightPlanView = this.$element.find(STRIP_VIEW_SELECTORS.FLIGHT_PLAN);
+
+        return this;
+    }
+
+    /**
+     * Create event handlers
+     *
+     * Should be run once only on instantiation
+     *
+     * @for StripViewModel
+     * @method _setupHandlers
+     * @chainable
+     */
+    _setupHandlers() {
+        this.$element.on('click', this._onClickHandler);
+        this.$element.on('dblclick', this._onDoubleClickHandler);
+
+        return this;
+    }
+
+    /**
+     * Set the layout with the correct data
+     *
+     * This method will be run on instantiation to set up the view for the first time
+     * and will also run on every subsequent re-render.
+     *
+     * We do not selectively render, if one property has changed
+     * (as determined by `_shouldUpdate()`) we re-draw everything in one shot.
+     *
+     * @for StripViewModel
+     * @method _layout
+     * @chainable
+     */
+    _layout() {
+        this.$element.addClass(this._categoryClassName);
+
+        return this;
+    }
+
+    /**
+     * Update teh view with new data
+     *
+     * This method will be run on instantiation to initialize the view with data,
+     * and will be run again any time updatable data has changed.
+     *
+     * After instantiation, this method should only be run after `._shouldUpdate()`
+     * has returned true.
+     *
+     * @for StripViewModel
+     * @method _render
+     * @chainable
+     */
+    _redraw() {
+        this.$callsignView.text(this._callsign);
+        this.$transponderView.text(this._transponder);
+        this.$aircraftTypeView.text(this._aircraftType);
+        this.$assignedAltitudeView.text(this._assignedAltitude);
+        this.$flightPlanAltitudeView.text(this._flightPlanAltitude);
+        this.$departureAirportView.text(this._departureAirport);
```

Comment:
the `.toUpperCase()` should actually live elsewhere (`AircraftModel.getViewModel()`), but I smell what you're stepping in 😉